Readme.md

```diff
@@ -7,7 +7,7 @@
 ## Example
 
 ```js
-var snippet = require('snippet');
+var snippet = require('segmentio-snippet');
```

Comment:
I'd just as soon republish this as `@segment/snippet` and deprecate this prefixed package via `npm deprecate`